src/configurator.ts

```diff
@@ -119,6 +120,57 @@ export class Configurator {
 
     core.addPath(toolPath);
   }
+
+  validate() {
+    if (this.name === "") {
```

Comment:
General nit: I think these can be `null` or whitespace under circumstances

Reply:
I think it can be `if(this.name)`, which should check for `null`, `undefined`, and `""`.
Good catch!